test/integration/smoke-testing/test_search.js

```diff
@@ -13,7 +13,7 @@ const tap = require('tap');
 const test = tap.test;
 
 // Set test url through environment variable
-const rootUrl = process.env.ROOT_URL || 'http://localhost:8333';
+const rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
```

Comment:
Is this change intentional?

Reply:
Yes.  All other tests in this suite point to scratch.ly by default.  This one pointed to localhost by default, meaning that if you don't put in a url, most tests will point to staging and these would fail unless you have a build running locally.